diff --git c/chrome/browser/browser_process_impl.cc w/chrome/browser/browser_process_impl.cc
index 681ad0d1e178926969cb88ddf53bb100ed6ba604..d193a91ee503c617f51fd29c5d9a1c891da415c3 100644
--- c/chrome/browser/browser_process_impl.cc
+++ w/chrome/browser/browser_process_impl.cc
@@ -282,7 +282,7 @@ void OnLocalStatePrefsLoaded();
 #endif
 
 #if BUILDFLAG(IS_MAC)
-#include "components/os_crypt/async/browser/keychain_key_provider.h"
+#include "maho/browser/os_crypt/maho_file_key_provider_mac.h"
 #endif
 
 #if BUILDFLAG(SAFE_BROWSING_AVAILABLE)
@@ -1661,10 +1661,12 @@ void BrowserProcessImpl::PreMainMessageLoopRun() {
 #endif  // BUILDFLAG(IS_POSIX) && !BUILDFLAG(IS_MAC) && !BUILDFLAG(IS_LINUX)
 
 #if BUILDFLAG(IS_MAC)
-  if (base::FeatureList::IsEnabled(features::kUseKeychainKeyProvider)) {
+  {
+    base::FilePath user_data_dir;
+    base::PathService::Get(chrome::DIR_USER_DATA, &user_data_dir);
     providers.emplace_back(std::make_pair(
-        /*precedence=*/10u,
-        std::make_unique<os_crypt_async::KeychainKeyProvider>()));
+        /*precedence=*/15u,
+        std::make_unique<maho::MahoFileKeyProvider>(user_data_dir)));
   }
 #endif  // BUILDFLAG(IS_MAC)
 
