diff --git c/chrome/browser/chrome_browser_main.cc w/chrome/browser/chrome_browser_main.cc
index 00fe10f55ffb3e0e983812df4c6c99742e2d84bc..cf39267604c81e281a2e722c7332c103d21a0fa2 100644
--- c/chrome/browser/chrome_browser_main.cc
+++ w/chrome/browser/chrome_browser_main.cc
@@ -318,6 +318,8 @@
 #include "chrome/browser/optimization_guide/chrome_browser_main_extra_parts_optimization_guide.h"
 #endif
 
+#include "maho/browser/maho_browser_main_extra_parts.h"
+
 // Separate per-platform blocks specifically for chrome_browser_main code. Put
 // other per-platform includes in the appropriate section above.
 #if BUILDFLAG(IS_WIN)
@@ -859,6 +861,8 @@ std::unique_ptr<content::BrowserMainParts> ChromeBrowserMainParts::Create(
       std::make_unique<ChromeBrowserMainExtraPartsOptimizationGuide>());
 #endif
 
+  main_parts->AddParts(std::make_unique<MahoBrowserMainExtraParts>());
+
   // Always add ChromeBrowserMainExtraPartsGpu last to make sure
   // GpuDataManager initialization could pick up about:flags settings.
   main_parts->AddParts(std::make_unique<ChromeBrowserMainExtraPartsGpu>());
