diff --git a/components/password_manager_strings.grdp b/components/password_manager_strings.grdp
index b03d0b9bd6539e8cda4d47b86ae5cab680b1e6a0..03771f023e16fb06676f14d79403021930f4b8b6 100644
--- a/components/password_manager_strings.grdp
+++ b/components/password_manager_strings.grdp
@@ -49,7 +49,7 @@
   </if>
   <if expr="not _google_chrome">
     <message name="IDS_PASSWORD_MANAGER_LEAK_HELP_MESSAGE" desc="The text is shown in a popup that explains to the user why they are seeing a warning about leaked passwords. This is a stand-alone string and the only content of the popup dialog.">
-      When you use a password, Chromium warns you if it has been published online. When doing this, your passwords and usernames are encrypted, so they can’t be read by anyone, including Google.
+      When you use a password, Maho warns you if it has been published online. When doing this, your passwords and usernames are encrypted, so they can’t be read by anyone, including Google.
     </message>
   </if>
   <message name="IDS_MANAGE_PASSWORDS_AUTO_SIGNIN_TITLE" desc="The title of the auto-signin toast.">
@@ -228,7 +228,7 @@
       Password Manager needs access to MacOS Keychain
     </message>
     <message name="IDS_PASSWORD_MANAGER_RELAUNCH_CHROME_BUBBLE_DESCRIPTION_NON_BRANDED" desc="Text that appears in the description of the relaunch Chromium bubble.">
-      To use Password Manager with macOS Keychain, relaunch Chromium and allow Keychain access. Your tabs will reopen after relaunching.
+      To use Password Manager with macOS Keychain, relaunch Maho and allow Keychain access. Your tabs will reopen after relaunching.
     </message>
   </if>
   <if expr="is_linux">
@@ -242,7 +242,7 @@
       Password Manager needs more access
     </message>
     <message name="IDS_PASSWORD_MANAGER_RELAUNCH_CHROME_BUBBLE_DESCRIPTION_LINUX_NON_BRANDED" desc="Text that appears in the description of the relaunch Chromium bubble.">
-      To use Password Manager with your operating system, relaunch Chromium and allow access to your computer's password manager. Your tabs will reopen after relaunching.
+      To use Password Manager with your operating system, relaunch Maho and allow access to your computer's password manager. Your tabs will reopen after relaunching.
     </message>
   </if>
   <if expr="is_win or is_macosx or is_chromeos">
@@ -268,7 +268,7 @@
       Relaunch Chrome
     </message>
     <message name="IDS_PASSWORD_MANAGER_RELAUNCH_CHROME_BUBBLE_ACCEPT_BUTTON_NON_BRANDED" desc="Text that appears in the accept button of the relaunch Chromium bubble.">
-      Relaunch Chromium
+      Relaunch Maho
     </message>
     <if expr="use_titlecase">
       <message name="IDS_PASSWORD_MANAGER_RELAUNCH_CHROME_BUBBLE_CANCEL_BUTTON" desc="Text that appears in the cancel button of the relaunch Chrome bubble.">
