diff --git a/components/reset_password_strings.grdp b/components/reset_password_strings.grdp
index 5f58c26466f0c0465d3b3edd715b59d28957d393..8b4828856aa5daee6d6ad35126d63655280d8cfa 100644
--- a/components/reset_password_strings.grdp
+++ b/components/reset_password_strings.grdp
@@ -40,10 +40,10 @@
   </if>
   <if expr="not _google_chrome">
      <message name="IDS_RESET_PASSWORD_EXPLANATION_PARAGRAPH" desc="The primary explanatory paragraph for the reset password page when the user manually navigates to this page.">
-      Chromium recommends resetting your password if you reused it on other sites.
+      Maho recommends resetting your password if you reused it on other sites.
     </message>
     <message name="IDS_RESET_PASSWORD_EXPLANATION_PARAGRAPH_WITH_ORG_NAME" desc="The primary explanatory paragraph for the reset password page when the user manually navigates to this page.">
-      Chromium recommends resetting your <ph name="BEGIN_BOLD">&lt;strong&gt;</ph><ph name="ORG_NAME">$1<ex>Google</ex></ph><ph name="END_BOLD">&lt;/strong&gt;</ph> password if you reused it on other sites.
+      Maho recommends resetting your <ph name="BEGIN_BOLD">&lt;strong&gt;</ph><ph name="ORG_NAME">$1<ex>Google</ex></ph><ph name="END_BOLD">&lt;/strong&gt;</ph> password if you reused it on other sites.
     </message>
   </if>
 </grit-part>
