diff --git c/components/affiliations/core/browser/affiliation_service_impl.cc w/components/affiliations/core/browser/affiliation_service_impl.cc
index 30c8cbe5ea13cd4f5f09b39c50de96c19a0b340d..b68dd07116f1ccabbf1e0c78932560e22d88c911 100644
--- c/components/affiliations/core/browser/affiliation_service_impl.cc
+++ w/components/affiliations/core/browser/affiliation_service_impl.cc
@@ -239,6 +239,7 @@ void AffiliationServiceImpl::Init(
 
 void AffiliationServiceImpl::Shutdown() {
   DCHECK_CALLED_ON_VALID_SEQUENCE(sequence_checker_);
+  prefetcher_.Shutdown();
   if (backend_) {
     backend_task_runner_->DeleteSoon(FROM_HERE, std::move(backend_));
   }
