diff --git a/chrome/common/webui_url_constants.cc b/chrome/common/webui_url_constants.cc
index dd37ab85cec115729953e06b23534b8c9d5bcc9e..54bcfcb9dbef235cfdeeda8edc1ab49af31984c3 100644
--- a/chrome/common/webui_url_constants.cc
+++ b/chrome/common/webui_url_constants.cc
@@ -15,6 +15,7 @@
 #include "components/optimization_guide/optimization_guide_internals/webui/url_constants.h"
 #include "components/password_manager/content/common/web_ui_constants.h"
 #include "components/safe_browsing/core/common/web_ui_constants.h"
+#include "maho/components/constants/webui_url_constants.h"
 #include "device/vr/buildflags/buildflags.h"
 #include "extensions/buildflags/buildflags.h"
 #include "third_party/blink/public/common/chrome_debug_urls.h"
@@ -149,6 +150,14 @@ base::span<const base::cstring_view> ChromeURLHosts() {
       history_clusters_internals::kChromeUIHistoryClustersInternalsHost,
       kChromeUIInterstitialHost,
       kChromeUILocalStateHost,
+      maho::kMahoAIHost,
+      maho::kMahoBoostHost,
+      maho::kMahoLiveFoldersHost,
+      maho::kMahoSettingsHost,
+      maho::kMahoSidebarHost,
+      maho::kMahoSidebarLegacyHost,
+      maho::kMahoSyncHost,
+      maho::kMahoTestHost,
 #if !BUILDFLAG(IS_ANDROID)
       kChromeUIManagementHost,
 #endif
