diff --git c/components/autofill/content/renderer/password_autofill_agent.cc w/components/autofill/content/renderer/password_autofill_agent.cc
index fac0b1f6c7c3c027cf3fb84fd96d8f1c24768723..f5f23e440f1de17b41092a786f1de18daa563571 100644
--- c/components/autofill/content/renderer/password_autofill_agent.cc
+++ w/components/autofill/content/renderer/password_autofill_agent.cc
@@ -869,12 +869,17 @@
     const std::u16string& password,
     base::OnceCallback<void(bool)> callback) {
   auto focused_element = last_queried_element().DynamicTo<WebInputElement>();
-  if (!focused_element && render_frame() && render_frame()->GetWebFrame()) {
-    focused_element = render_frame()->GetWebFrame()->GetDocument().FocusedElement().DynamicTo<WebInputElement>();
+  if (auto* frame = unsafe_render_frame(); !focused_element && frame) {
+    if (auto* web_frame = frame->GetWebFrame()) {
+      auto document = web_frame->GetDocument();
+      if (document) {
+        focused_element = document.FocusedElement().DynamicTo<WebInputElement>();
+      }
+    }
   }
   if (!focused_element && !web_input_to_password_info_.empty()) {
     for (const auto& [field_ref, info] : web_input_to_password_info_) {
-      auto el = field_ref.GetField().DynamicTo<WebInputElement>();
+      auto el = GetFormControlByRendererId(field_ref).DynamicTo<WebInputElement>();
       if (el && IsElementEditable(el)) {
         focused_element = el;
         break;
