diff --git c/components/password_manager/core/browser/password_store/password_store.cc w/components/password_manager/core/browser/password_store/password_store.cc
index 547e70f4317d23f1614ea7f698ce50bd9302e414..11797aef030c2767c6db95a8a1c2d4327b555497 100644
--- c/components/password_manager/core/browser/password_store/password_store.cc
+++ w/components/password_manager/core/browser/password_store/password_store.cc
@@ -316,9 +316,14 @@ void PasswordStore::GetLogins(const PasswordFormDigest& form,
   }
 
   if (post_init_callback_) {
-    post_init_callback_ = std::move(post_init_callback_)
-                              .Then(base::BindOnce(&PasswordStore::GetLogins,
-                                                   this, form, consumer));
+    post_init_callback_ =
+        std::move(post_init_callback_)
+            .Then(base::BindOnce(
+                static_cast<void (PasswordStore::*)(
+                    const PasswordFormDigest&,
+                    base::WeakPtr<PasswordStoreConsumer>)>(
+                    &PasswordStore::GetLogins),
+                this, form, consumer));
     return;
   }
 
@@ -327,6 +332,43 @@ void PasswordStore::GetLogins(const PasswordFormDigest& form,
                            consumer));
 }
 
+void PasswordStore::GetLogins(
+    const PasswordFormDigest& form,
+    const PasswordFillRequestContext& context,
+    base::WeakPtr<PasswordStoreConsumer> consumer) {
+  DCHECK(main_task_runner_->RunsTasksInCurrentSequence());
+  if (!backend_) {
+    return;
+  }
+  if (post_init_callback_) {
+    post_init_callback_ =
+        std::move(post_init_callback_)
+            .Then(base::BindOnce(
+                static_cast<void (PasswordStore::*)(
+                    const PasswordFormDigest&,
+                    const PasswordFillRequestContext&,
+                    base::WeakPtr<PasswordStoreConsumer>)>(
+                    &PasswordStore::GetLogins),
+                this, form, context, consumer));
+    return;
+  }
+  backend_->GetGroupedMatchingLoginsAsync(
+      form, context,
+      base::BindOnce(&ConsumerReplyConverter, consumer,
+                     base::RetainedRef(this)));
+}
+
+void PasswordStore::ResolvePasswordFill(
+    const PasswordFillRequestContext& context,
+    const PasswordFillSelection& selection,
+    PasswordFillResolver resolver) {
+  if (!backend_) {
+    std::move(resolver).Run(std::nullopt);
+    return;
+  }
+  backend_->ResolvePasswordFill(context, selection, std::move(resolver));
+}
+
 void PasswordStore::GetAutofillableLogins(
     base::WeakPtr<PasswordStoreConsumer> consumer) {
   DCHECK(main_task_runner_->RunsTasksInCurrentSequence());
