diff --git c/components/password_manager/core/browser/password_store/password_store_backend_metrics_recorder.cc w/components/password_manager/core/browser/password_store/password_store_backend_metrics_recorder.cc
index 7f17a6edc1bdaf180cd17abe6be5b9b9c68036f6..a4289c87decc33db245ed20cc1b1b3ccbdfbadeb 100644
--- c/components/password_manager/core/browser/password_store/password_store_backend_metrics_recorder.cc
+++ w/components/password_manager/core/browser/password_store/password_store_backend_metrics_recorder.cc
@@ -3,6 +3,7 @@
 // found in the LICENSE file.
 
 #include "components/password_manager/core/browser/password_store/password_store_backend_metrics_recorder.h"
+#include "components/password_manager/core/browser/password_store/password_store_backend.h"
 
 #include <utility>
 #include <variant>
@@ -15,6 +16,20 @@
 
 namespace password_manager {
 
+void PasswordStoreBackend::GetGroupedMatchingLoginsAsync(
+    const PasswordFormDigest& form_digest,
+    const PasswordFillRequestContext& context,
+    BackendLoginsOrErrorReply callback) {
+  GetGroupedMatchingLoginsAsync(form_digest, std::move(callback));
+}
+
+void PasswordStoreBackend::ResolvePasswordFill(
+    const PasswordFillRequestContext& context,
+    const PasswordFillSelection& selection,
+    PasswordFillResolver resolver) {
+  std::move(resolver).Run(std::nullopt);
+}
+
 namespace {
 constexpr char kMetricPrefix[] = "PasswordManager.PasswordStore";
 
