diff --git c/components/password_manager/core/browser/password_store/password_store.h w/components/password_manager/core/browser/password_store/password_store.h
index 8e139f66eb5a56e1d3fa9773afb4498d43a14ac4..a25e94d370829ce7900702a7698131bfa1319b3c 100644
--- c/components/password_manager/core/browser/password_store/password_store.h
+++ w/components/password_manager/core/browser/password_store/password_store.h
@@ -88,6 +88,14 @@ class PasswordStore : public PasswordStoreInterface {
       base::OnceClosure completion = base::NullCallback()) override;
   void GetLogins(const PasswordFormDigest& form,
                  base::WeakPtr<PasswordStoreConsumer> consumer) override;
+  void GetLogins(
+      const PasswordFormDigest& form,
+      const PasswordFillRequestContext& context,
+      base::WeakPtr<PasswordStoreConsumer> consumer) override;
+  void ResolvePasswordFill(
+      const PasswordFillRequestContext& context,
+      const PasswordFillSelection& selection,
+      PasswordFillResolver resolver) override;
   void GetAutofillableLogins(
       base::WeakPtr<PasswordStoreConsumer> consumer) override;
   void GetAllLogins(base::WeakPtr<PasswordStoreConsumer> consumer) override;
