diff --git c/chrome/common/webui_url_constants.cc w/chrome/common/webui_url_constants.cc
index 9c32ebdbbd83fa0fd545762158e0871750ee661e..37d70a444b31d8dd72b7e78ef952110114f57b8b 100644
--- c/chrome/common/webui_url_constants.cc
+++ w/chrome/common/webui_url_constants.cc
@@ -15,6 +15,7 @@
 #include "components/optimization_guide/optimization_guide_internals/webui/url_constants.h"
 #include "components/password_manager/content/common/web_ui_constants.h"
 #include "components/safe_browsing/core/common/web_ui_constants.h"
+#include "maho/components/constants/webui_url_constants.h"
 #include "device/vr/buildflags/buildflags.h"
 #include "extensions/buildflags/buildflags.h"
 #include "third_party/blink/public/common/chrome_debug_urls.h"
@@ -151,6 +152,14 @@ base::span<const base::cstring_view> ChromeURLHosts() {
       kChromeUIIwaDevHost,
 #endif
       kChromeUILocalStateHost,
+      maho::kMahoAIHost,
+      maho::kMahoBoostHost,
+      maho::kMahoLiveFoldersHost,
+      maho::kMahoSettingsHost,
+      maho::kMahoSidebarHost,
+      maho::kMahoSidebarLegacyHost,
+      maho::kMahoSyncHost,
+      maho::kMahoTestHost,
 #if !BUILDFLAG(IS_ANDROID)
       kChromeUIManagementHost,
 #endif
