diff --git a/components/password_manager/core/browser/password_store/password_store_consumer.cc b/components/password_manager/core/browser/password_store/password_store_consumer.cc
index 2ba39c30106464fc1b02e11f24bdd496b5643b2e..36bad39338c6be9ed9825a5859afb4b5f919a8de 100644
--- a/components/password_manager/core/browser/password_store/password_store_consumer.cc
+++ b/components/password_manager/core/browser/password_store/password_store_consumer.cc
@@ -30,6 +30,20 @@ void PasswordStoreConsumer::OnGetPasswordStoreResultsOrErrorFrom(
                      std::move(results_or_error))));
 }
 
+void PasswordStoreInterface::GetLogins(
+    const PasswordFormDigest& form,
+    const PasswordFillRequestContext& context,
+    base::WeakPtr<PasswordStoreConsumer> consumer) {
+  GetLogins(form, std::move(consumer));
+}
+
+void PasswordStoreInterface::ResolvePasswordFill(
+    const PasswordFillRequestContext& context,
+    const PasswordFillSelection& selection,
+    PasswordFillResolver resolver) {
+  std::move(resolver).Run(std::nullopt);
+}
+
 void PasswordStoreConsumer::OnGetPasswordStoreResults(
     std::vector<std::unique_ptr<PasswordForm>> results) {}
 
