diff --git a/components/password_manager/core/browser/password_store/password_store.h b/components/password_manager/core/browser/password_store/password_store.h
index 667a69c4567c44165b8a7f9e5f3eca8e0053bcda..880cffc4e315bdf8102b0a98118caff55b5f1cf3 100644
--- a/components/password_manager/core/browser/password_store/password_store.h
+++ b/components/password_manager/core/browser/password_store/password_store.h
@@ -92,6 +92,14 @@ class PasswordStore : public PasswordStoreInterface {
       base::OnceClosure completion = base::NullCallback()) override;
   void GetLogins(const PasswordFormDigest& form,
                  base::WeakPtr<PasswordStoreConsumer> consumer) override;
+  void GetLogins(
+      const PasswordFormDigest& form,
+      const PasswordFillRequestContext& context,
+      base::WeakPtr<PasswordStoreConsumer> consumer) override;
+  void ResolvePasswordFill(
+      const PasswordFillRequestContext& context,
+      const PasswordFillSelection& selection,
+      PasswordFillResolver resolver) override;
   void GetAutofillableLogins(
       base::WeakPtr<PasswordStoreConsumer> consumer) override;
   void GetAllLogins(base::WeakPtr<PasswordStoreConsumer> consumer) override;
