diff --git c/components/password_manager/content/browser/content_password_manager_driver.cc w/components/password_manager/content/browser/content_password_manager_driver.cc
index 45520108b9a9e6dbb93d9436497d9f37d77b776e..405884ae105e92f8a90b954a0625630bebba46f4 100644
--- c/components/password_manager/content/browser/content_password_manager_driver.cc
+++ w/components/password_manager/content/browser/content_password_manager_driver.cc
@@ -118,6 +118,7 @@ void ContentPasswordManagerDriver::BindPendingReceiver(
 }
 
 void ContentPasswordManagerDriver::DidNavigate() {
+  RotatePasswordManagerDocumentToken();
   if (!IsRenderFrameHostSupported()) {
     password_manager_receiver_.reset();
   }
