diff --git c/chrome/browser/ui/customize_chrome/side_panel_controller_base.cc w/chrome/browser/ui/customize_chrome/side_panel_controller_base.cc
index 17334384ffbe8e25244614df24b250779e87c6b6..78fb1887f8473d970fb1099eeb5c18acd9054609 100644
--- c/chrome/browser/ui/customize_chrome/side_panel_controller_base.cc
+++ w/chrome/browser/ui/customize_chrome/side_panel_controller_base.cc
@@ -39,7 +39,7 @@ SidePanelControllerBase::SidePanelControllerBase(tabs::TabInterface& tab)
   will_discard_contents_callback_subscription_ =
       tab_->RegisterWillDiscardContents(
           base::BindRepeating(&SidePanelControllerBase::WillDiscardContents,
-                              base::Unretained(this)));
+                              weak_ptr_factory_.GetWeakPtr()));
 }
 
 SidePanelControllerBase::~SidePanelControllerBase() = default;
@@ -128,8 +128,15 @@ void SidePanelControllerBase::CreateAndRegisterEntry() {
   }
   auto entry = std::make_unique<SidePanelEntry>(
       SidePanelEntry::Key(SidePanelEntry::Id::kCustomizeChrome),
-      base::BindRepeating(&SidePanelControllerBase::CreateCustomizeChromeView,
-                          base::Unretained(this)),
+      base::BindRepeating(
+          [](base::WeakPtr<SidePanelControllerBase> controller,
+             SidePanelEntryScope& scope) -> SidePanelNativeView {
+            if (!controller) {
+              return nullptr;
+            }
+            return controller->CreateCustomizeChromeView(scope);
+          },
+          weak_ptr_factory_.GetWeakPtr()),
       /*default_content_width_callback=*/base::NullCallback());
 #if BUILDFLAG(IS_ANDROID)
   entry->SetProperty(
