diff --git a/chrome/browser/ui/cocoa/browser_window_command_handler.mm b/chrome/browser/ui/cocoa/browser_window_command_handler.mm
index 52c9fa13295a6126aa0c427cad43a0a3b73525b3..26fcf54421d43c48cc95dce0bc67fe00f93b9d8d 100644
--- a/chrome/browser/ui/cocoa/browser_window_command_handler.mm
+++ b/chrome/browser/ui/cocoa/browser_window_command_handler.mm
@@ -10,10 +10,13 @@
 #include "base/strings/sys_string_conversions.h"
 #include "chrome/app/chrome_command_ids.h"
 #import "chrome/browser/app_controller_mac.h"
+#include "chrome/browser/ui/browser.h"
 #include "chrome/browser/profiles/profile.h"
 #include "chrome/browser/ui/browser_commands.h"
-#include "chrome/browser/ui/browser_finder.h"
 #include "chrome/browser/ui/browser_window.h"
+#include "chrome/browser/ui/browser_window/public/browser_window_interface.h"
+#include "chrome/browser/ui/browser_window/public/global_browser_collection.h"
+#include "maho/browser/ui/views/sidebar/maho_sidebar_prefs.h"
 #include "components/remote_cocoa/app_shim/native_widget_ns_window_bridge.h"
 #include "components/remote_cocoa/common/native_widget_ns_window_host.mojom.h"
 #include "content/public/browser/web_contents.h"
@@ -61,6 +64,20 @@ remote_cocoa::NativeWidgetNSWindowBridge* FindBridgeForSender(
   return bridge;
 }
 
+Browser* FindBrowserForSender(id sender, NSWindow* window) {
+  NSWindow* targetWindow = window;
+  if ([sender respondsToSelector:@selector(window)]) {
+    targetWindow = [sender window];
+  }
+  BrowserWindowInterface* browser_window =
+      GlobalBrowserCollection::GetInstance()->FindBrowserWithWindow(
+          gfx::NativeWindow(targetWindow));
+  DCHECK(browser_window);
+  Browser* browser = browser_window->GetBrowserForMigrationOnly();
+  DCHECK(browser);
+  return browser;
+}
+
 }  // namespace
 
 @implementation BrowserWindowCommandHandler
@@ -108,6 +125,11 @@ remote_cocoa::NativeWidgetNSWindowBridge* FindBridgeForSender(
 - (void)commandDispatch:(id)sender window:(NSWindow*)window {
   DCHECK(sender);
   int command = [sender tag];
+  if (command == IDC_SAVE_PAGE) {
+    maho::sidebar_prefs::ToggleSidebarPanelExpanded(
+        FindBrowserForSender(sender, window)->profile()->GetPrefs());
+    return;
+  }
   bool was_executed = false;
   FindBridgeForSender(sender, window)
       ->host()
@@ -141,6 +163,12 @@ remote_cocoa::NativeWidgetNSWindowBridge* FindBridgeForSender(
     modifierFlags &= ~NSEventModifierFlagCommand;
   }
 
+  if (command == IDC_SAVE_PAGE) {
+    maho::sidebar_prefs::ToggleSidebarPanelExpanded(
+        FindBrowserForSender(sender, window)->profile()->GetPrefs());
+    return;
+  }
+
   bool was_executed = false;
   FindBridgeForSender(sender, window)
       ->host()
