diff --git a/chrome/browser/ui/ash/arc/arc_open_url_delegate_impl.cc b/chrome/browser/ui/ash/arc/arc_open_url_delegate_impl.cc
index ec9e208b707c21b7ab8d239bb3b65a249bc23f6f..f800e62abc27e6721fa5de27cb98099abee73b56 100644
--- a/chrome/browser/ui/ash/arc/arc_open_url_delegate_impl.cc
+++ b/chrome/browser/ui/ash/arc/arc_open_url_delegate_impl.cc
@@ -38,6 +38,8 @@
 #include "chrome/browser/ui/ash/shelf/app_service/app_service_app_window_arc_tracker.h"
 #include "chrome/browser/ui/ash/shelf/app_service/app_service_app_window_shelf_controller.h"
 #include "chrome/browser/ui/ash/shelf/chrome_shelf_controller.h"
+#include "chrome/browser/ui/browser_window/public/browser_window_interface_iterator.h"
+#include "chrome/browser/ui/chrome_pages.h"
 #include "chrome/browser/web_applications/web_app_helpers.h"
 #include "chrome/browser/web_applications/web_app_utils.h"
 #include "chrome/browser/webshare/prepare_directory_task.h"
@@ -337,8 +339,9 @@ void ArcOpenUrlDelegateImpl::OpenChromePageFromArc(ChromePage page) {
 
   if (auto it = kBrowserSettingsMap.find(page);
       it != kBrowserSettingsMap.end()) {
-    OpenUrlFromArc(
-        GURL(ash::chrome_urls::kChromeUISettingsURL).Resolve(it->second));
+    if (auto* browser = GetLastActiveBrowserWindowInterfaceWithAnyProfile()) {
+      chrome::ShowSettingsSubPage(browser, it->second);
+    }
     return;
   }
 
