diff --git c/components/new_or_sad_tab_strings.grdp w/components/new_or_sad_tab_strings.grdp
index 12ba9f38eb526033c5f5bb548a3d6b0cd6f7e3ca..0460026dbba7b1dc2656fa0c16be89a7d2640ce9 100644
--- c/components/new_or_sad_tab_strings.grdp
+++ w/components/new_or_sad_tab_strings.grdp
@@ -88,7 +88,7 @@
       </if>
       <if expr="not _google_chrome">
         <message name="IDS_SAD_TAB_RELOAD_RESTART_BROWSER" desc="One of the bullet points displayed on the web page if a reload failed to fix the issue, advising the user to restart Chromium." formatter_data="android_java">
-          Restart Chromium
+          Restart Maho
         </message>
       </if>
       <if expr="is_android or is_ios">
@@ -173,7 +173,7 @@
       </if>
       <if expr="not _google_chrome">
         <message name="IDS_NEW_TAB_OTR_NOT_SAVED" desc="Bullet points listing data that are not saved in the Incognito mode. 'Browsing history' means a history of visited websites. 'Cookies and site data' refers to data saved by websites on the user's device (e.g. sign-in state, preferences, etc.). 'Information entered in forms' refers to names, addresses, passwords etc. that users enter into forms on the web." formatter_data="android_java">
-          Chromium <ph name="BEGIN_EMPHASIS">&lt;em&gt;</ph>won’t save<ph name="END_EMPHASIS">&lt;/em&gt;</ph>:
+          Maho <ph name="BEGIN_EMPHASIS">&lt;em&gt;</ph>won’t save<ph name="END_EMPHASIS">&lt;/em&gt;</ph>:
           <ph name="BEGIN_LIST">&lt;ul&gt;</ph>
             <ph name="LIST_ITEM">&lt;li&gt;</ph>Your browsing history
             <ph name="LIST_ITEM">&lt;li&gt;</ph>Cookies and site data
