diff --git c/components/password_manager_strings.grdp w/components/password_manager_strings.grdp
index 5094e1943a0e9b2fd339ff478dd65cbfb715bb08..e3791d6006b6e4a0b052dba7ba13ce733571109e 100644
--- c/components/password_manager_strings.grdp
+++ w/components/password_manager_strings.grdp
@@ -52,7 +52,7 @@
   </if>
   <if expr="not _google_chrome">
     <message name="IDS_PASSWORD_MANAGER_LEAK_HELP_MESSAGE" desc="The text is shown in a popup that explains to the user why they are seeing a warning about leaked passwords. This is a stand-alone string and the only content of the popup dialog.">
-      When you use a password, Chromium warns you if it has been published online. When doing this, your passwords and usernames are encrypted, so they can’t be read by anyone, including Google.
+      When you use a password, Maho warns you if it has been published online. When doing this, your passwords and usernames are encrypted, so they can’t be read by anyone, including Google.
     </message>
   </if>
   <message name="IDS_MANAGE_PASSWORDS_AUTO_SIGNIN_TITLE" desc="The title of the auto-signin toast.">
@@ -237,7 +237,7 @@
       Password Manager needs access to MacOS Keychain
     </message>
     <message name="IDS_PASSWORD_MANAGER_RELAUNCH_CHROME_BUBBLE_DESCRIPTION_NON_BRANDED" desc="Text that appears in the description of the relaunch Chromium bubble.">
-      To use Password Manager with macOS Keychain, relaunch Chromium and allow Keychain access. Your tabs will reopen after relaunching.
+      To use Password Manager with macOS Keychain, relaunch Maho and allow Keychain access. Your tabs will reopen after relaunching.
     </message>
   </if>
   <if expr="is_linux">
@@ -251,7 +251,7 @@
       Password Manager needs more access
     </message>
     <message name="IDS_PASSWORD_MANAGER_RELAUNCH_CHROME_BUBBLE_DESCRIPTION_LINUX_NON_BRANDED" desc="Text that appears in the description of the relaunch Chromium bubble.">
-      To use Password Manager with your operating system, relaunch Chromium and allow access to your computer's password manager. Your tabs will reopen after relaunching.
+      To use Password Manager with your operating system, relaunch Maho and allow access to your computer's password manager. Your tabs will reopen after relaunching.
     </message>
   </if>
   <if expr="is_win or is_macosx or is_chromeos">
@@ -277,7 +277,7 @@
       Relaunch Chrome
     </message>
     <message name="IDS_PASSWORD_MANAGER_RELAUNCH_CHROME_BUBBLE_ACCEPT_BUTTON_NON_BRANDED" desc="Text that appears in the accept button of the relaunch Chromium bubble.">
-      Relaunch Chromium
+      Relaunch Maho
     </message>
     <if expr="use_titlecase">
       <message name="IDS_PASSWORD_MANAGER_RELAUNCH_CHROME_BUBBLE_CANCEL_BUTTON" desc="Text that appears in the cancel button of the relaunch Chrome bubble.">
