diff --git c/chrome/browser/ui/views/passwords/password_bubble_view_base.cc w/chrome/browser/ui/views/passwords/password_bubble_view_base.cc
index 8f65ff4b24b841797295d02bf8a7b093cf83c962..a1dc13c08c5ac865f942b97d6c98ba2079edb054 100644
--- c/chrome/browser/ui/views/passwords/password_bubble_view_base.cc
+++ w/chrome/browser/ui/views/passwords/password_bubble_view_base.cc
@@ -248,7 +248,8 @@ PasswordBubbleViewBase::PasswordBubbleViewBase(
 PasswordBubbleViewBase::~PasswordBubbleViewBase() {
   CHECK(this != g_manage_passwords_bubble_);
   // It is possible in tests for |browser_| not to exist.
-  if (browser_ && BrowserActions::From(browser_)->root_action_item()) {
+  if (browser_ && BrowserActions::From(browser_) &&
+      BrowserActions::From(browser_)->root_action_item()) {
     if (auto* passwords_action_item = actions::ActionManager::Get().FindAction(
             kActionShowPasswordsBubbleOrPage,
             BrowserActions::From(browser_)->root_action_item())) {
