diff --git c/chrome/browser/ui/views/frame/multi_contents_view.cc w/chrome/browser/ui/views/frame/multi_contents_view.cc
index c8fdc7f0c725ea1d1fa73e580a85a743a62ad92c..8be14dc216157a50a44ceaa34dd3f14cbb1f77a0 100644
--- c/chrome/browser/ui/views/frame/multi_contents_view.cc
+++ w/chrome/browser/ui/views/frame/multi_contents_view.cc
@@ -493,7 +493,7 @@
 
 void MultiContentsView::OnActorOverlayFocused(views::WebView* web_view) {
   if (IsInSplitView() && GetWidget()->IsVisible()) {
-    for (auto* contents_container_view : contents_container_views_) {
+    for (ContentsContainerView* contents_container_view : contents_container_views_) {
       if (contents_container_view->actor_overlay_web_view() == web_view &&
           contents_container_view != GetActiveContentsContainerView()) {
         return delegate_->WebContentsFocused(
@@ -505,7 +505,7 @@
 
 void MultiContentsView::OnNtpFooterFocused(views::WebView* web_view) {
   if (IsInSplitView() && GetWidget()->IsVisible()) {
-    for (auto* contents_container_view : contents_container_views_) {
+    for (ContentsContainerView* contents_container_view : contents_container_views_) {
       if (contents_container_view->new_tab_footer_view() == web_view &&
           contents_container_view != GetActiveContentsContainerView()) {
         return delegate_->WebContentsFocused(
@@ -600,7 +600,8 @@
         maho_extra_child == contents_separators_.trailing_separator ||
         maho_extra_child == contents_separators_.top_separator ||
         maho_extra_child == contents_separators_.corner_separator ||
-        base::Contains(contents_container_views_, maho_extra_child)) {
+        std::ranges::contains(contents_container_views_, maho_extra_child,
+                              [](const auto& container) { return container.get(); })) {
       continue;
     }
     LOG(WARNING) << "[MAHO_GRAD] LayoutManager extra_child rect="
