diff --git c/chrome/browser/ui/cocoa/accelerators_cocoa.mm w/chrome/browser/ui/cocoa/accelerators_cocoa.mm
index 12d6be60e250efa15872027b297baab899e250c9..b48284287dd452f48eb45b92feacbe03bc4f6012 100644
--- c/chrome/browser/ui/cocoa/accelerators_cocoa.mm
+++ w/chrome/browser/ui/cocoa/accelerators_cocoa.mm
@@ -89,6 +89,7 @@ const struct AcceleratorMapping {
      ui::EF_COMMAND_DOWN | ui::EF_SHIFT_DOWN, ui::VKEY_V},
     {IDC_CONTENT_CONTEXT_SELECTALL, ui::EF_COMMAND_DOWN, ui::VKEY_A},
     {IDC_FOCUS_SEARCH, ui::EF_COMMAND_DOWN | ui::EF_ALT_DOWN, ui::VKEY_F},
+    {IDC_MAHO_AI_PANEL, ui::EF_COMMAND_DOWN, ui::VKEY_E},
     {IDC_FIND_NEXT, ui::EF_COMMAND_DOWN, ui::VKEY_G},
     {IDC_FIND_PREVIOUS, ui::EF_COMMAND_DOWN | ui::EF_SHIFT_DOWN, ui::VKEY_G},
     {IDC_STOP, ui::EF_COMMAND_DOWN, ui::VKEY_OEM_PERIOD},
