diff --git a/components/password_manager/content/browser/content_password_manager_driver.cc b/components/password_manager/content/browser/content_password_manager_driver.cc
index 28f7beb14e7afe569edc530f9fcfbee950a19ea0..0e92159f733bd890d283a0da3ce735aac74e2be0 100644
--- a/components/password_manager/content/browser/content_password_manager_driver.cc
+++ b/components/password_manager/content/browser/content_password_manager_driver.cc
@@ -166,6 +166,7 @@ void ContentPasswordManagerDriver::BindPendingReceiver(
 }
 
 void ContentPasswordManagerDriver::DidNavigate() {
+  RotatePasswordManagerDocumentToken();
   if (!IsRenderFrameHostSupported(render_frame_host_)) {
     password_manager_receiver_.reset();
   }
