diff --git a/components/password_manager/core/browser/password_form_digest.h b/components/password_manager/core/browser/password_form_digest.h
index c49cdb761c357a429ff9d739683398429c6498b8..02e52c0d2cfd0d21b07a39c7d8f533bf40ff2f42 100644
--- a/components/password_manager/core/browser/password_form_digest.h
+++ b/components/password_manager/core/browser/password_form_digest.h
@@ -8,6 +8,7 @@
 #include <string>
 
 #include "components/password_manager/core/browser/password_form.h"
+#include "maho/chromium_src/components/password_manager/core/browser/password_fill_request.h"
 #include "url/gurl.h"
 
 namespace autofill {
