diff --git a/components/password_manager/core/browser/password_form_manager.cc b/components/password_manager/core/browser/password_form_manager.cc
index 0aa08a1c1e74838a6166bfa9993f8a5854abe89f..d1265d112866e74e596d40af9e76d9cea4df494a 100644
--- a/components/password_manager/core/browser/password_form_manager.cc
+++ b/components/password_manager/core/browser/password_form_manager.cc
@@ -307,6 +307,8 @@ PasswordFormManager::PasswordFormManager(
   if (driver_) {
     driver_id_ = driver->GetId();
     cached_driver_frame_id_ = driver->GetFrameId();
+    form_fetcher_->SetPasswordFillRequestContext(
+        driver_->GetPasswordFillRequestContext());
   }
 
   metrics_recorder_->RecordFormSignature(
