diff --git a/chrome/browser/ui/views/passwords/password_bubble_view_base.cc b/chrome/browser/ui/views/passwords/password_bubble_view_base.cc
index 8f91c0e858daf968ab2cf46692ff8868b0b19792..c7b68c3c91cea55104c01b352ab282971eefe128 100644
--- a/chrome/browser/ui/views/passwords/password_bubble_view_base.cc
+++ b/chrome/browser/ui/views/passwords/password_bubble_view_base.cc
@@ -245,12 +245,14 @@ PasswordBubbleViewBase::PasswordBubbleViewBase(
 PasswordBubbleViewBase::~PasswordBubbleViewBase() {
   CHECK(this != g_manage_passwords_bubble_);
   // It is possible in tests for |browser_| not to exist.
-  if (browser_) {
+  if (browser_ && browser_->browser_actions() &&
+      browser_->browser_actions()->root_action_item()) {
     auto* passwords_action_item = actions::ActionManager::Get().FindAction(
         kActionShowPasswordsBubbleOrPage,
         browser_->browser_actions()->root_action_item());
-    CHECK(passwords_action_item);
-    passwords_action_item->SetIsShowingBubble(false);
+    if (passwords_action_item) {
+      passwords_action_item->SetIsShowingBubble(false);
+    }
   }
 }
 
