diff --git c/chrome/app/password_manager_ui_strings.grdp w/chrome/app/password_manager_ui_strings.grdp
index c3ed41189b5906ca25d6c4b342fb459f11f4b34b..f364b94af281ffe38eaa66de489931d1e1b09c00 100644
--- c/chrome/app/password_manager_ui_strings.grdp
+++ w/chrome/app/password_manager_ui_strings.grdp
@@ -896,7 +896,7 @@
         Password Manager needs access to MacOS Keychain
       </message>
       <message name="IDS_PASSWORD_MANAGER_UI_RELAUNCH_CHROME_PROMO_CARD_DESCRIPTION" desc="Description for not branded Relaunch ChrChromiumome promo card.">
-        To use Password Manager with macOS Keychain, relaunch Chromium and allow Keychain access. Your tabs will reopen after relaunching.
+        To use Password Manager with macOS Keychain, relaunch Maho and allow Keychain access. Your tabs will reopen after relaunching.
       </message>
     </else>
     </if>
@@ -916,7 +916,7 @@
         Password Manager needs more access
       </message>
       <message name="IDS_PASSWORD_MANAGER_UI_RELAUNCH_CHROME_PROMO_CARD_DESCRIPTION_LINUX" desc="Description for the Relaunch Chromium promo card.">
-        To use Password Manager with your operating system, relaunch Chromium and allow access to your computer's password manager. Your tabs will reopen after relaunching.
+        To use Password Manager with your operating system, relaunch Maho and allow access to your computer's password manager. Your tabs will reopen after relaunching.
       </message>
     </else>
     </if>
@@ -930,7 +930,7 @@
     </then>
     <else>
       <message name="IDS_PASSWORD_MANAGER_UI_RELAUNCH_CHROME_PROMO_CARD_ACTION" desc="Action button for not branded Relaunch Chrome promo card.">
-        Relaunch Chromium
+        Relaunch Maho
       </message>
     </else>
     </if>
@@ -1028,7 +1028,7 @@
   </then>
   <else>
     <message name="IDS_PASSWORD_MANAGER_UI_PASSWORD_CHANGE_TOAST_SIGN_IN_TO_CONTINUE" desc="The description for the automated password change entry in Password Manager settings.">
-      You need to sign in before Chromium can finish this task
+      You need to sign in before Maho can finish this task
     </message>
   </else>
   </if>
@@ -1099,7 +1099,7 @@
   </then>
   <else>
     <message name="IDS_PASSWORD_MANAGER_UI_PASSWORD_CHANGE_SETTINGS_DESCRIPTION" desc="The description for the automated password change entry in Password Manager settings.">
-      If Chromium finds one of your passwords in a public data breach, it can offer to change your password for you if you're signed in
+      If Maho finds one of your passwords in a public data breach, it can offer to change your password for you if you're signed in
     </message>
   </else>
   </if>
@@ -1160,7 +1160,7 @@
   </then>
   <else>
     <message name="IDS_SETTINGS_PASSWORD_CHANGE_SUBLABEL_WITH_LEARN_MORE" desc="Sublabel for password change entry in settings. The description of the feature beneath the 'Automated password change' feature title. This and the bullets beneath this paragraph represent the full description of the feature. * 'public' does two things: 1) it provides distance between 'Chrome' and 'data breach', because we don’t want users to associate the breach with Chrome (Chrome didn’t have a data leak, 2) and related, it might reassure the user to know that Chrome is using public information to establish these lists of compromised passwords. * 'Google Password Manager can offer…': The user has to agree to have this feature update their password for them each time. So Google Password Manager is ‘offering’ to do something for the user.">
-      If Chromium finds one of your passwords in a public data breach, Google Password Manager can offer to change your password for you if you're signed in. <ph name="BEGIN_LINK">&lt;a href="$1" target="_blank"&gt;</ph>More about automated password change<ph name="END_LINK">&lt;/a&gt;</ph>
+      If Maho finds one of your passwords in a public data breach, Google Password Manager can offer to change your password for you if you're signed in. <ph name="BEGIN_LINK">&lt;a href="$1" target="_blank"&gt;</ph>More about automated password change<ph name="END_LINK">&lt;/a&gt;</ph>
     </message>
   </else>
   </if>
@@ -1181,7 +1181,7 @@
     </then>
     <else>
       <message name="IDS_PASSWORD_MANAGER_UI_USER_INTERVENTION_NEEDED_TITLE" desc="Title of a dialog shown when the automated password change stops because the website requires identity verification.">
-        Chromium needs help changing your password
+        Maho needs help changing your password
       </message>
     </else>
   </if>
