diff --git c/chrome/browser/ui/views/location_bar/BUILD.gn w/chrome/browser/ui/views/location_bar/BUILD.gn
index 76ff8c936f24938da7c7fcc51bc2ca2636d765cb..98a0fcbf81337f381e42f691765c488544d2683a 100644
--- c/chrome/browser/ui/views/location_bar/BUILD.gn
+++ w/chrome/browser/ui/views/location_bar/BUILD.gn
@@ -75,6 +75,7 @@ source_set("impl") {
     "//chrome/browser:defaults",
     "//chrome/browser/ui/dialogs",
     "//chrome/browser/ui/select_file_policy",
+    "//maho/browser:maho_location_bar_views",
   ]
 
   if (toolkit_views) {
@@ -322,6 +323,7 @@ source_set("unit_tests") {
   deps = [
     ":icon_label_bubble",
     ":location_bar",
+    "//maho/browser:maho_location_bar_views_unittests",
     "//chrome/app:generated_resources",
     "//chrome/test:test_support",
     "//components/omnibox/browser",
@@ -362,6 +364,8 @@ source_set("interactive_ui_tests") {
     "//chrome/browser/ui/views/toolbar",
     "//chrome/browser/ui/views/translate",
     "//chrome/browser/ui/views/web_apps/frame_toolbar:test_support",
+    "//maho/browser:maho_location_bar_views",
+    "//maho/browser:maho_location_bar_views_interactive_tests",
     "//chrome/browser/web_applications:web_applications_test_support",
     "//chrome/test:test_support",
     "//chrome/test:test_support_ui",
