diff --git a/chrome/browser/password_manager/chrome_password_manager_client.cc b/chrome/browser/password_manager/chrome_password_manager_client.cc
index f7b68f2c94eb14f39495fb251f81d3f3b0a782df..6ca40def40b845adb8a88b60f814fd72ae68153e 100644
--- a/chrome/browser/password_manager/chrome_password_manager_client.cc
+++ b/chrome/browser/password_manager/chrome_password_manager_client.cc
@@ -127,6 +127,7 @@
 #include "content/public/browser/web_contents.h"
 #include "content/public/common/content_switches.h"
 #include "extensions/buildflags/buildflags.h"
+#include "maho/browser/passwords/maho_password_provider_utils.h"
 #include "net/base/url_util.h"
 #include "net/cert/cert_status_flags.h"
 #include "services/metrics/public/cpp/metrics_utils.h"
@@ -269,6 +270,9 @@ ChromePasswordManagerClient::~ChromePasswordManagerClient() = default;
 
 bool ChromePasswordManagerClient::IsSavingAndFillingEnabled(
     const GURL& url) const {
+  if (!maho::passwords::ShouldEnableChromiumPasswordManager()) {
+    return false;
+  }
   if (base::CommandLine::ForCurrentProcess()->HasSwitch(
           switches::kEnableAutomation)) {
     // Disable the password saving UI for automated tests. It obscures the
@@ -284,6 +288,9 @@ bool ChromePasswordManagerClient::IsSavingAndFillingEnabled(
 }
 
 bool ChromePasswordManagerClient::IsFillingEnabled(const GURL& url) const {
+  if (!maho::passwords::ShouldEnableChromiumPasswordManager()) {
+    return false;
+  }
   const Profile* profile =
       Profile::FromBrowserContext(web_contents()->GetBrowserContext());
   // Guest profiles don't have PasswordStore at all, so filling should be
