diff --git c/components/password_manager/core/browser/form_fetcher_impl.h w/components/password_manager/core/browser/form_fetcher_impl.h
index c229797f2a3dfb4d86235ae79db2ba324d1f177b..1a0bb51627d9aacc6bf7c48a00d1afbc6573fbf6 100644
--- c/components/password_manager/core/browser/form_fetcher_impl.h
+++ w/components/password_manager/core/browser/form_fetcher_impl.h
@@ -48,6 +48,8 @@ class FormFetcherImpl : public FormFetcher,
   void AddConsumer(FormFetcher::Consumer* consumer) override;
   void RemoveConsumer(FormFetcher::Consumer* consumer) override;
   void Fetch() override;
+  void SetPasswordFillRequestContext(
+      const PasswordFillRequestContext& context) override;
   State GetState() const override;
   const std::vector<InteractionsStats>& GetInteractionsStats() const override;
   base::span<const StoredCredential> GetInsecureCredentials() const override;
@@ -97,6 +99,7 @@ class FormFetcherImpl : public FormFetcher,
 
   // PasswordStore results will be fetched for this description.
   const PasswordFormDigest form_digest_;
+  std::optional<PasswordFillRequestContext> fill_request_context_;
 
   // Client used to obtain a CredentialFilter.
   const raw_ptr<PasswordManagerClient> client_;
