diff --git c/components/password_manager/core/browser/password_form_manager.cc w/components/password_manager/core/browser/password_form_manager.cc
index 3602205e9ced2f5229ed71e28c03b757c17d2468..0cf7c26a6ed04332c4aaec88273f5344466f0b2b 100644
--- c/components/password_manager/core/browser/password_form_manager.cc
+++ w/components/password_manager/core/browser/password_form_manager.cc
@@ -311,6 +311,8 @@ PasswordFormManager::PasswordFormManager(
   if (driver_) {
     driver_id_ = driver->GetId();
     cached_driver_frame_id_ = driver->GetFrameId();
+    form_fetcher_->SetPasswordFillRequestContext(
+        driver_->GetPasswordFillRequestContext());
   }
 
   metrics_recorder_->RecordFormSignature(
