diff --git i/components/password_manager/core/browser/password_manager_driver.cc w/components/password_manager/core/browser/password_manager_driver.cc
new file mode 100644
index 0000000000000000000000000000000000000000..a4098be38ab0ef005e9544782b53fbb1c2fe1692
--- /dev/null
+++ w/components/password_manager/core/browser/password_manager_driver.cc
@@ -0,0 +1,19 @@
+#include "components/password_manager/core/browser/password_manager_driver.h"
+
+#include <utility>
+
+namespace password_manager {
+
+void PasswordManagerDriver::FillSuggestion(
+    const PasswordFillRequestContext& context,
+    const std::u16string& username,
+    const std::u16string& password,
+    base::OnceCallback<void(bool)> success_callback) {
+  if (!IsPasswordFillRequestContextCurrent(context)) {
+    std::move(success_callback).Run(false);
+    return;
+  }
+  FillSuggestion(username, password, std::move(success_callback));
+}
+
+}
