diff --git a/chrome/browser/ui/views/side_panel/side_panel_coordinator.h b/chrome/browser/ui/views/side_panel/side_panel_coordinator.h
index 507522f5d38ac34dfa6feadca7a7cc6bd09885e6..53d0a894b7d7fbff50766bc1c50e2d8049729c8f 100644
--- a/chrome/browser/ui/views/side_panel/side_panel_coordinator.h
+++ b/chrome/browser/ui/views/side_panel/side_panel_coordinator.h
@@ -10,6 +10,7 @@
 
 #include "base/callback_list.h"
 #include "base/memory/raw_ptr.h"
+#include "base/memory/weak_ptr.h"
 #include "base/observer_list.h"
 #include "base/time/time.h"
 #include "chrome/browser/ui/side_panel/side_panel_entry.h"
@@ -117,6 +118,8 @@ class SidePanelCoordinator final : public SidePanelUIBase,
       side_panel_toolbar_pinning_controller_;
 
   ui::ScopedUnownedUserData<SidePanelCoordinator> scoped_unowned_user_data_;
+
+  base::WeakPtrFactory<SidePanelCoordinator> weak_ptr_factory_{this};
 };
 
 #endif  // CHROME_BROWSER_UI_VIEWS_SIDE_PANEL_SIDE_PANEL_COORDINATOR_H_
