diff --git a/chrome/browser/ui/accelerator_table.cc b/chrome/browser/ui/accelerator_table.cc
index 51837acdf5a13e6ae75e163943d977c85c6c7fd1..d035ddfa462e0f8ad660023a09954112b42452ab 100644
--- a/chrome/browser/ui/accelerator_table.cc
+++ b/chrome/browser/ui/accelerator_table.cc
@@ -63,8 +63,6 @@ const AcceleratorMapping kAcceleratorMap[] = {
     {ui::VKEY_W, ui::EF_SHIFT_DOWN | ui::EF_PLATFORM_ACCELERATOR,
      IDC_CLOSE_WINDOW},
     {ui::VKEY_F, ui::EF_PLATFORM_ACCELERATOR, IDC_FIND},
-    {ui::VKEY_A, ui::EF_SHIFT_DOWN | ui::EF_PLATFORM_ACCELERATOR,
-     IDC_TAB_SEARCH},
     {ui::VKEY_G, ui::EF_PLATFORM_ACCELERATOR, IDC_FIND_NEXT},
     {ui::VKEY_G, ui::EF_SHIFT_DOWN | ui::EF_PLATFORM_ACCELERATOR,
      IDC_FIND_PREVIOUS},
@@ -237,9 +235,8 @@ const AcceleratorMapping kAcceleratorMap[] = {
      IDC_FOCUS_INACTIVE_POPUP_FOR_ACCESSIBILITY},
     {ui::VKEY_D, ui::EF_ALT_DOWN, IDC_FOCUS_LOCATION},
     {ui::VKEY_E, ui::EF_CONTROL_DOWN, IDC_FOCUS_SEARCH},
-    {ui::VKEY_K, ui::EF_CONTROL_DOWN, IDC_FOCUS_SEARCH},
     {ui::VKEY_R, ui::EF_SHIFT_DOWN | ui::EF_ALT_DOWN,
-     IDC_SHOW_READING_MODE_KEYBOARD},
+      IDC_SHOW_READING_MODE_KEYBOARD},
     {ui::VKEY_T, ui::EF_SHIFT_DOWN | ui::EF_ALT_DOWN, IDC_FOCUS_TOOLBAR},
     {ui::VKEY_RIGHT, ui::EF_ALT_DOWN, IDC_FORWARD},
     {ui::VKEY_RIGHT, ui::EF_ALTGR_DOWN, IDC_FORWARD},
