diff --git a/ui/views/cocoa/drag_drop_client_mac.mm b/ui/views/cocoa/drag_drop_client_mac.mm
index 29e4f18792b8c67084b93e3ba6837bc7dcd924bb..84082009c2e5cabbe7f599a3dfdced47b6b56202 100644
--- a/ui/views/cocoa/drag_drop_client_mac.mm
+++ b/ui/views/cocoa/drag_drop_client_mac.mm
@@ -5,6 +5,7 @@
 #import "ui/views/cocoa/drag_drop_client_mac.h"
 
 #include "base/mac/mac_util.h"
+#include "ui/gfx/geometry/vector2d.h"
 #include "base/run_loop.h"
 #include "base/strings/sys_string_conversions.h"
 #import "components/remote_cocoa/app_shim/bridged_content_view.h"
@@ -90,7 +91,7 @@ void DragDropClientMac::StartDragAndDrop(
     }
   }
 
-  [bridge_->ns_view() beginDraggingSessionWithItems:drag_items
+  dragging_session_ = [bridge_->ns_view() beginDraggingSessionWithItems:drag_items
                                               event:event
                                              source:bridge_->ns_view()];
 
@@ -130,6 +131,7 @@ NSDragOperation DragDropClientMac::Drop(id<NSDraggingInfo> sender) {
 void DragDropClientMac::EndDrag() {
   exchange_data_.reset();
   is_drag_source_ = false;
+  dragging_session_ = nil;
 
   // Allow a test to invoke EndDrag() without spinning the nested run loop.
   if (!quit_closure_.is_null()) {
@@ -162,4 +164,35 @@ gfx::Point DragDropClientMac::LocationInView(
   }
   return LocationInView(point);
 }
+
+void DragDropClientMac::UpdateDragImage(const gfx::ImageSkia& image,
+                                        const gfx::Vector2d& offset) {
+  if (!dragging_session_ || image.isNull()) {
+    return;
+  }
+
+  NSWindow* window = bridge_->ns_window();
+  NSPoint mouse_location = window.mouseLocationOutsideOfEventStream;
+  NSImage* ns_image = gfx::NSImageFromImageSkia(image);
+  CGFloat width = ns_image.size.width;
+  CGFloat height = ns_image.size.height;
+  NSRect dragging_frame = NSMakeRect(mouse_location.x - offset.x(),
+                                     mouse_location.y - (height - offset.y()),
+                                     width, height);
+
+  [dragging_session_ enumerateDraggingItemsWithOptions:NSDraggingItemEnumerationConcurrent
+                                               forView:bridge_->ns_view()
+                                               classes:@[ NSPasteboardItem.class ]
+                                         searchOptions:@{}
+                                            usingBlock:^(NSDraggingItem* item, NSInteger idx, BOOL* stop) {
+    if (idx == 0) {
+      [item setDraggingFrame:dragging_frame contents:ns_image];
+    } else {
+      [item setDraggingFrame:NSMakeRect(0, 0, 1, 1) contents:nil];
+    }
+  }];
+
+  dragging_session_.draggingFormation = NSDraggingFormationDefault;
+}
+
 }  // namespace views
